packages/plugins/other/visitor-plugin-common/src/avoid-optionals.ts

```diff
@@ -1,21 +1,29 @@
-import { AvoidOptionalsConfig } from './types.js';
+import { AvoidOptionalsConfig, NormalizedAvoidOptionalsConfig } from './types.js';
 
-export const DEFAULT_AVOID_OPTIONALS: AvoidOptionalsConfig = {
+export const DEFAULT_AVOID_OPTIONALS: NormalizedAvoidOptionalsConfig = {
```

Comment:
`NormalizedAvoidOptionalsConfig` is similar to `AvoidOptionalsConfig` but all the fields are non-optional, making it much easier to code with